### PR TITLE
Update over info

### PR DESCRIPTION
**Pull request by sebaslogen**

### CODE REVIEW

File: app/employees/server/lib/qers.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const currentQers = [
   { name: "Roelf-Jan de Vries", handle:"roelfjan", web: "http://www.roelf-jandevries.nl", tumblr: "roelfjandevries" },
   { name: "Roos Luteijn", handle:"roos", imageStatic: "anonymous.jpg", imageAnimated: "anonymous.jpg" },
   { name: "Sahan Aydin" , handle:"sahan" },
-  { name: "Sebastian Lebato Genco", handle: "sebastian", imageStatic: "anonymous.jpg" },
+  { name: "Sebastián Lobato Genco", handle: "sebas" },
```
Comment (lukasvan3l) [inline app/employees/server/lib/qers.js]: Sorry about the misspelling of your name!
The "handle" property is the prefix of your e-mailaddress. So it's supposed to be sebastian. Also in the labels and filename of your image. Change those and I'll merge!
p.s. I added the alias sebas@q42.nl / sebas@q42.com so you can use those if you prefer. But your gsuite username is still sebastian@q42.nl.

Comment (sebaslogen) [inline app/employees/server/lib/qers.js]: Don't worry about the misspelling. Thanks for the email alias 💯

**Comment by sebaslogen** [comment]
Fixed code review comments and squashed in a single commit.

**Comment by lukasvan3l** [comment]
awesome!! I'll try to deploy tomorrow

**Pull request merged**